scripts/setup/upgrade-postgresql

```diff
@@ -76,7 +76,6 @@ fi
     ZULIP_CONF="${TEMP_CONF_DIR}/zulip.conf"
     crudini --set "$ZULIP_CONF" postgresql version "$UPGRADE_TO"
     crudini --set "$ZULIP_CONF" machine puppet_classes zulip::profile::base,zulip::postgresql_base
-    touch "/usr/share/postgresql/$UPGRADE_TO/pgroonga_setup.sql.applied"
```

Comment:
I think the second paragraph of the commit message is missing a "not".

Reply:
I don't think so?
```quote
upgrade-postgresql: Do not touch pgroonga_setup.sql.applied.

This step is incorrect, since c8ec3dfcf6dd -- the file must contain
the version that was configured.  As such, touching the file
accomplishes nothing.

We remove it, since this will result in the next `zulip-puppet-apply`
running `ALTER EXTENSION pgroonga UPDATE`, which is at worst a no-op
-- and may well be required if the PostgreSQL upgrade has also
resulted in an upgrade of pgroonga.
```

Can you show where you think the missing negation is?

Reply:
Moot because I've changed the commit and commit message.